src/MySql/Sql/CreateAuditTrigger.php

```diff
@@ -246,9 +289,8 @@ private function createInsertStatement($rowState)
       $values .= sprintf('%s.`%s`', $rowState, $column['column_name']);
     }
 
-    $insertStatement = sprintf('insert into `%s`.`%s`(%s)
-values(%s);
-',
+    $insertStatement = sprintf("insert into `%s`.`%s`(%s)
```

Comment:
This adds 2 lines. Just pass $sqlStatement as parameter just as method skipStatement.